bindings/python/pinocchio/robot_wrapper.py

```diff
@@ -5,33 +5,27 @@
 from . import pinocchio_pywrap_default as pin
 from . import utils
 from .shortcuts import (
-    buildModelsFromMJCF,
     buildModelsFromSdf,
+    buildModelsFromMJCF,
     buildModelsFromUrdf,
     createDatas,
 )
 
 
 class RobotWrapper(object):
     @staticmethod
-    def BuildFromURDF(
-        filename, package_dirs=None, root_joint=None, verbose=False, meshLoader=None
-    ):
+    def BuildFromURDF(filename, *args, **kwargs):
         robot = RobotWrapper()
-        robot.initFromURDF(filename, package_dirs, root_joint, verbose, meshLoader)
+
+        robot.initFromURDF(filename, kwargs)
```

Comment:
```suggestion
        robot.initFromURDF(filename, *args, **kwargs)
```

You must add the * (tuple) and ** (dict) to turn args and kwargs into parameters.